### ISSUE TITLE
Email: Comments on WCAG3

### ISSUE THREAD

**Issue by jspellman** [w3c/silver#414]
[Comment from Email](https://lists.w3.org/Archives/Public/public-agwg-comments/2021Feb/0001.html) 
Comments on WCAG 3 by Robert G. Martinengo
1/30/21

With all due respect to the Silver working group, please just stop for a 
moment. I have been in the accessibility field for over twenty years and 
have come to understand that after a certain point, continued emphasis 
on accessibility as a separate topic only serves to further distance 
people with disability from the mainstream. The WCAG is at that point, 
and it's time to make a decision.

I reviewed the various posted materials on WCAG 3. The amount of time, 
effort, and care that went in to them is impressive, evident, and 
admirable. But as it stands, WCAG 3 is bound to fail to live up to the 
lofty goals set for it. This is not a prediction, but an observation 
based on experience.

The most important thing that needs to happen with WCAG is removing the 
words 'accessibility' and 'guidelines'. Replace them with something like 
'building codes': "...sets of regulations governing the design, 
construction, alteration and maintenance of structures. They specify the 
minimum requirements to adequately safeguard the health, safety and 
welfare of building occupants."
https://www.fema.gov/emergency-managers/risk-management/earthquake/seismic-building-codes 
<https://www.fema.gov/emergency-managers/risk-management/earthquake/seismic-building-codes> 

Adopting the building code model for the web is not just a matter of 
semantics – it's a seismic shift in the way accessibility is 
conceptualized. Not to overstretch parallels with construction, but if 
mark-up languages, scripts, style sheets, etc are the raw materials, web 
building codes are the specifications for how these parts can be 
assembled in to web sites and applications that safeguard access for all 
users (authoring tools and user agents fit nicely in to this framework 
as well). The main levels of web building codes should be 'personal', 
'commercial', and 'government'. The main point is, accessibility is 
woven into the codes, not as a separate topic but simply as the way 
things are done.

Here's some good news: if the W3C adopts this approach, all of the great 
resources and experience generated by WCAG, and the terrific research 
and work done on v.3 will definitely not go to waste. Instead, WCAG will 
become the backbone of the new web building codes, ensuring 
accessibility becomes fundamental to the web and second-nature to those 
who design, build, use, and pay for web sites and apps. There will still 
be specific testing criteria, just as there are for building codes, but 
compliance with web building codes will be more holistic than testing 
for accessibility alone.

On the surface, it's a classic Catch-22: the only way to achieve 
sustainable accessibility is to stop talking about accessibility. Yes, 
there will be plenty of challenges transitioning from accessibility 
guidelines to web building codes, but the difference is between 
continuing to head down a dead end road, versus merging with the 
mainstream freeway of web development to ensure that people with 
disabilities will be in the front seat.

Thanks for your time, and I'm happy to respond to any comments or questions.
Robert G. Martinengo

**Comment by jspellman** [comment]
Second email from Bob in response to Alastair: 
Hi Alastair,
Thanks so much for your feedback – I appreciate your thoughtful reply.
I think I'm really trying to make two related points.
The first is that, to grow, WCAG definitely needs to be rebranded, which is
ironic, because 'accessibility' should always apply to everyone, right? If
the guidelines had been called 'disability access guidelines', that would
have made it clear who the target population was. Instead, 'we', the a11y
folks, have our own special subset of people in mind when we use the word
accessibility, and I think that is limiting and ableist. If the web is for
everyone, and it should be, then all web standards should respect the
variety of everyone's needs and abilities.
As you say, the web is a super-dynamic environment consisting of an
ever-evolving profusion of products and products (whew! it's tiring just
thinking about it). But that is exactly why we should be attempting to
embed the core principles of accessibility in to the web's DNA by
repositioning WCAG at the heart of webs standards. I realize I didn't
express that point very well before – I am proposing the WCAG become the
key foundation of a broader 'web building code' (sort of like when Neo
subsumes Agent Smith). I think it would have a liberating effect on the web
ecosystem and lead to better alignment between the various components of
the web ecosystem.
Which brings me to the second point – the impact and value of WCAG itself.
How do we measure impact? One way, obviously, is by how many organizations
adopt it and how many governments point to it in their regulations. What is
really being adopted are the compliance criteria (a quick search brings up
plenty of a11y testing that promises automated testing for WCAG
compliance). This is well and good, but the number of lawsuits against
major companies for inaccessible websites would seem to indicate there is
still a long way to go (we should be measuring the difference we are making
for people with disabilities accessing the web).
So, how could a 'building code' model help make such a difference? Well,
just as physical buildings undergo inspection before getting a permit to
open, why not apply the same principle to websites?
Let me illustrate with a hypothetical: A new pizza chain readies its
website for the public. It does thorough user and a11y testing, and when
they think they have it right and it meets code, they apply for a permit to
begin operation (you can envision various scenarios of testing, variances,
etc., and it would be up to regulators how to handle the permit process,
etc., but the basics are pretty straight forward).

Now is the time for the W3C to step back and take a look at the big
picture. They can take the blue pill and stick with good, old WCAG, or take
a chance on the red pill, and chart an exciting new future for a truly
accessible and inclusive web.
Peace out,
Robert

**Comment by cwadamsoforacle** [comment]
Proposed response:
This content has been updated with the release of the latest draft of WCAG 3, which can be found here:  [W3C Accessibility Guidelines (WCAG) 3.0](https://www.w3.org/TR/wcag-3.0/).  The Accessibility Guidelines Working Group attempted to address these concerns in this latest draft.  Should you feel that these issues remain in the current draft, please log a new issue here:  [New Issue · w3c/wcag3](https://github.com/w3c/wcag3/issues/new).
